sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx

```diff
@@ -81,9 +108,7 @@ const NetworksSwitcher: FC = () => {
 	return !isL2 ? (
 		<Container onClick={openChainModal}>
 			<StyledButton noOutline size="sm">
-				<PrefixIcon>
-					<EthereumIcon width={20} height={14} />
-				</PrefixIcon>
+				{activeChain && networkIcon(activeChain.name)}
```

Comment:
If you decide to do the above, then we would have to change it here too.

Reply:
Thanks. Will make the change.